Assets/Scripts/ResultScript.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ResultScript : MonoBehaviour
+{
+    bool next = false;
+    [SerializeField] GameObject result_ui;
+    [SerializeField] GameObject next_ui;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Return) && !next){
```

Comment:
```suggestion
        if (Input.GetKeyDown(KeyCode.Return)){
```
nextの値は不変のため、nextは要らない変数な気がするのですが、なぜnextを使ったのでしょうか？

Reply:
一度しか処理が行われないようにするためにnextをtrueにするのを忘れていました

Reply:
複数回実行されても結果は特に変わらないように思うので、nextを消してしまってもいいかと思います。

Reply:
その案で修正させていただきます